multisrc/overrides/mangathemesia/legacyscans/src/LegacyScans.kt

```diff
@@ -0,0 +1,7 @@
+package eu.kanade.tachiyomi.extension.fr.legacyscans
+
+import eu.kanade.tachiyomi.multisrc.mangathemesia.MangaThemesia
+import java.text.SimpleDateFormat
+import java.util.Locale
+
+class LegacyScans : MangaThemesia("Legacy Scans", "https://legacy-scans.com", "fr", dateFormat = SimpleDateFormat("MMMM d, yyyy", Locale.FRENCH))
```

Comment:
```suggestion
class LegacyScans : MangaThemesia("Legacy Scans", "https://legacy-scans.com", "fr", dateFormat = SimpleDateFormat("MMMM d, yyyy", Locale.FRENCH)) {
    override val id = 8947802555328550956
}
```

Keep the old ID so users will not have to migrate after the update.